Sofa/GUI/Component/src/sofa/gui/component/performer/ConstraintAttachBodyPerformer.inl

```diff
@@ -89,21 +90,19 @@ bool ConstraintAttachBodyPerformer<DataTypes>::startPartial(const BodyPicked& pi
     m_mstate1 = dynamic_cast<MouseContainer*>(this->m_interactor->getMouseContainer());
     m_mstate2 = mstateCollision;
 
-    type::Vec3d point1;
-    type::Vec3d point2;
-
     using sofa::component::constraint::lagrangian::model::BilateralLagrangianConstraint;
 
 
-
-    this->m_interactionObject = sofa::core::objectmodel::New<BilateralLagrangianConstraint<sofa::defaulttype::Vec3Types> >(m_mstate1, m_mstate2);
-    auto* bconstraint = dynamic_cast< BilateralLagrangianConstraint< sofa::defaulttype::Vec3Types >* >(this->m_interactionObject.get());
+    this->m_interactionObject = sofa::core::objectmodel::New<BilateralLagrangianConstraint<DataTypes> >(m_mstate1, m_mstate2);
+    auto* bconstraint = dynamic_cast< BilateralLagrangianConstraint< DataTypes >* >(this->m_interactionObject.get());
 
```

Comment:
It might be good to enable the data `d_keepOrientDiff` to avoid having the beam forced to be oriented with the world frame.